bundle/manifests/konveyor-forklift-operator.v99.0.0.clusterserviceversion.yaml

```diff
@@ -174,7 +174,7 @@ spec:
     The Konveyor Forklift Operator fully manages the deployment and life cycle of Forklift on OpenShift.
 
     If you experience any issues or have feature requests, please [file an issue in our Github repository](https://github.com/konveyor/forklift-operator/issues)
-  keywords: ['migration']
+  keywords: ["migration", "forklift"]
```

Comment:
Can you also add konveyor as a keyword, please?

Reply:
done